### PR TITLE
Add a service to get the stake from a public key

### PR DESCRIPTION
**Pull request by ureeves**
Resolves: #464

### CODE REVIEW

File: contracts/stake/src/lib.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ pub const MATURITY: u32 = 2 * EPOCH;
 pub const VALIDITY: u32 = 56 * EPOCH;
 
 /// The minimum amount of Dusk one can stake.
-pub const MINIMUM_STAKE: u64 = 5_000;
+pub const MINIMUM_STAKE: u64 = 5_000_000;
```
Comment (herr-seppia) [inline contracts/stake/src/lib.rs:29]: 5M dusk or 5M nanodusk?

**Review by herr-seppia — COMMENTED** [review]
I'm not sure about the minimum amount of staked dusk

File: contracts/stake/src/lib.rs (hunk 1 above)
Comment (ureeves) [reply contracts/stake/src/lib.rs:29]: It should be 5k M *micro*dusk. Thanks for making me look twice. It needs three more zeros.

Comment (herr-seppia) [reply contracts/stake/src/lib.rs]: Perfect, so it needs a change in the comment too

**Review by vlopes11 — APPROVED** [review]
LGTM, we will update after to use bls public keys

**Pull request merged**